.github/workflows/urls-check.yml

```diff
@@ -0,0 +1,22 @@
+name: urls-check
+on:
+  schedule:
+    - cron: '0 4 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
+  workflow_dispatch:
+jobs:
+  urls-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up configuration and secrets
+        run: |
+          printf '[user]\n\tname = Cockpit Project\n\temail=cockpituous@gmail.com\n' > ~/.gitconfig
+          echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run urls-check action
+        run: |
+          make bots
```

Comment:
That has the same problem, though?

Reply:
yeah, I fix it with the other commit. Should I do this the right way in the first commit?

Reply:
Ah, yes please, less confusing.

Reply:
done